src/components/filters/DateFilter/utils/tests/OptionUtils.spec.ts

```diff
@@ -0,0 +1,47 @@
+// (C) 2019 GoodData Corporation
+import {
+    isSameDate,
+    getTimezoneOffset,
+    isNegativeGTMTime,
+    getDateFilterOptionGranularity,
+} from "../OptionUtils";
+import { absoluteFormFilter, relativePresetFilter } from "../Translations/tests/fixtures";
+
+describe("optionUtils", () => {
+    describe("getDateFilterOptionGranularity", () => {
+        it("should return ", () => {
+            expect(getDateFilterOptionGranularity(absoluteFormFilter)).toEqual(undefined);
+            expect(getDateFilterOptionGranularity(relativePresetFilter)).toEqual("GDC.time.date");
+        });
+    });
+
+    describe("isSameDate", () => {
+        it.each([
+            [true, "2019-11-01", "2019-11-01"],
+            [false, "2019-11-01", "2019-11-03"],
+            [true, new Date("2019-11-11"), new Date("2019-11-11")],
+            [false, new Date("2019-11-11"), new Date("2019-11-10")],
+        ])("should return value is %s", (expected: boolean, from: Date | string, to: Date | string) => {
+            expect(isSameDate(from, to)).toEqual(expected);
+        });
+    });
+
+    it("should getTimezoneOffset return timezone offset correctly", () => {
+        Date.prototype.getTimezoneOffset = () => {
```

Comment:
Here you overwrite the prototype method and it probably will be rewritten for all other tests. I think it would be safer to restore mocked implementation in case of some other test will original behavior...